frontend/src/FullPageRoutes/RecordPage.tsx

```diff
@@ -17,32 +18,41 @@ const useStyles = makeStyles({
 })
 
 function sendVideo(
-  blob: Nullable<Blob>,
+  movie: Nullable<vd.Movie>,
   receiver: NullableId,
   history: History
-) {
-  if (blob) {
-    const reader = new FileReader()
-    reader.readAsDataURL(blob)
-    reader.onload = () => {
-      request({
-        path: "/api/videos/",
-        method: "post",
-        body: {
-          receiver: receiver,
-          data: reader.result,
-        },
-        parser: parseIdentity,
-      }).then(() => history.goBack())
-    }
+): void {
+  if (movie) {
+    movie
+      .record({
+        frameRate: 60,
+      })
+      .then((blob) => {
+        const reader = new FileReader()
+        reader.readAsDataURL(blob)
+        reader.onload = () => {
+          request({
+            path: "/api/videos/",
+            method: "post",
+            body: {
+              receiver: receiver,
+              data: reader.result,
+            },
+            parser: parseIdentity,
+          }).then(() => history.goBack())
+        }
+      })
   }
 }
 
 export default function RecordPage(): JSX.Element {
-  const [videoBlob, setVideoBlob] = React.useState<Nullable<Blob>>(null)
+  const [, setVideoBlob] = React.useState<Nullable<Blob>>(null)
```

Comment:
Why isn't the state value destructured here? What use is a state if the state data isn't referenced anywhere?

Reply:
Taking a second look at this, it looks like that state is redundant